### ISSUE TITLE
Parent Dashboard not showing all sub-projects

### ISSUE THREAD

**Issue by ronaldcs** [jgraichen/redmine_dashboard#103]
Hello,

I have a parent project called CIS with two sub-projects, CAS and CDS. In the Projects view, it shows fine:

![image](https://cloud.githubusercontent.com/assets/4138434/8806104/fdfc4660-2fa2-11e5-9886-4c7a6c7b632c.png)

However, in the CIS Dashboard view, it only shows one of the projects:

![image](https://cloud.githubusercontent.com/assets/4138434/8806128/1e30af34-2fa3-11e5-99f3-a101224c339a.png)

The CAS project does have issues in it:

![image](https://cloud.githubusercontent.com/assets/4138434/8806154/41b08ef2-2fa3-11e5-8c27-27c773409596.png)

Both sub-projects are on version 1.0 and the Include Sub-projects option is checked. Something I'm doing wrong?

Additional info:

```
Server: Ubuntu 14.04
Environment:
  Redmine version                3.0.4.stable
  Ruby version                   2.1.4-p265 (2014-10-27) [x86_64-linux]
  Rails version                  4.2.3
  Environment                    production
  Database adapter               Mysql2
Redmine plugins:
  redmine_bootstrap_kit          0.2-devel
  redmine_dashboard              2.5.0 -- NOTE: I actually downloaded and installed 2.6.1
  redmine_git_hosting            1.1.1
```
Thanks,

Ron

**Comment by jgraichen** [comment]
Probably same as #96?

**Comment by ronaldcs** [comment]
Possibly...but I just created an empty sub-project for CIS called Test and that doesn't show up either with 0 issues, like #96.

**Comment by jgraichen** [comment]
Fixed in 2.7.0. See #96.
